site/src/pages/WorkspaceSettingsPage/WorkspaceSettingsForm.tsx

```diff
@@ -54,16 +58,20 @@ export const WorkspaceSettingsForm: FC<{
     <HorizontalForm onSubmit={form.handleSubmit} data-testid="form">
       <FormSection
         title="Workspace Name"
-        description="Update the name of your workspace."
+        description="The name of your workspace."
```

Comment:
I might keep the old language here. The user already knows this is an input. Also, the other field (and any other fields we add in the future) are going to use imperative descriptions.